### DISCUSSION TITLE
taro 如何 更改 打包后 注入 html 的 script 的 publicPath

### DISCUSSION THREAD

**Discussion by weishenlan** [NervJS/taro#11889] [Q & A] [answered]
### 这个特性解决了什么问题？
类似 html-webpack-plugin 的publicPath属性，可以直接更改打包后 注入 html 的 script 的 publicPath。目前只能使用__webpack_public_path__去更改打包后的文件的public path.但被注入 html 的 script 的 publicPath没有被更改。

### 这个 API 长什么样？
提供一个类似html-webpack-plugin 的第taro生态包，用法相同

**Comment by Chen-jj** [comment, marked as answer]
@weishenlan 使用 webpackchain 去修改内置的 html-webpack-plugin 的参数？
